docs/API/v3/search-query-service-resource.md

```diff
@@ -0,0 +1,147 @@
+`q` is provided, all packages should be returned. This enables the "Browse" tab in the NuGet Visual Studio
+experience.
+
+The `skip` parameter defaults to 0.
+
+The `take` parameter should be an integer greater than zero. The server implementation may impose a maximum value.
+
+If `prerelease` is not provided, pre-release packages are excluded.
+
+The `semVerLevel` query parameter is used to opt-in to SemVer 2.0.0 packages. If this query parameter is excluded, only
+SemVer 1.0.0 compatible packages will be returned. If `semVerLevel=2.0.0` is provided, both SemVer 1.0.0 and SemVer 2.0.0
+compatible packages will be returned. See the [SemVer 2.0.0 protocol](https://github.com/NuGet/Home/wiki/Semver-2.0.0-Protocol)
+for more information.
+
+### Response
+
+The response is JSON document containing up to `take` search results. Search results are grouped by package ID.
+
+Search results returned by nuget.org looks something like this, when using `q=NuGet.Versioning&prerelease=false`:
+
+[!code-JSON [search-result.json](./_data/search-result.json)]
+
+The root JSON object has the following properties:
+
+Name      | Type             | Notes
+--------- | ---------------- | -----
+totalHits | integer          | Required: the total number of matches, disregarding `skip` and `take`
+data      | array of objects | Required: the search results matched by the request
+
+### Search result
+
+Each item in the `data` array is a JSON object comprised of a group of package versions sharing the same package ID.
+The object has the following properties:
+
+Name           | Type                       | Notes
+-------------- | -------------------------- | -----
+id             | string                     | Required: the ID of the matched package
+version        | string                     | Required: the full SemVer 2.0.0 version string of the package (could contain build metadata)
+description    | string                     | Optional
+versions       | array of objects           | Required: all of the versions of the package matching the `prerelease` parameter
+authors        | string or array of strings | Optional
+iconUrl        | string                     | Optional
+licenseUrl     | string                     | Optional
+owners         | string or array of strings | Optional
+projectUrl     | string                     | Optional
+registration   | string                     | Optional: the absolute URL to the associated [registration index](registration-base-url-resource.md#registration-index)
+summary        | string                     | Optional
+tags           | string or array of strings | Optional
+title          | string                     | Optional
+totalDownloads | integer                    | Optional: this value can be inferred by the sum of downloads in the `versions` array
+verified       | boolean                    | Optional: a JSON boolean indicating whether the package is [verified](../../reference/id-prefix-reservation.md)
```

Comment:
What is the value other implementors should use here?

Reply:
Hmm, great point. I think it should be excluded by other implementors but I'll talk to the team.

Reply:
Fixed.